src/aura/BGE_ConfigurationWizard/BGE_ConfigurationWizardHelper.js

```diff
@@ -63,64 +63,64 @@
-                templateMetadataView.errorMessage = templateMetadata.errorMessage;
-                templateMetadataView.pageHeader = templateMetadata.pageHeader;
-                templateMetadataView.pendingSave = templateMetadata.pendingSave;
-
-                if (!templateMetadataView.hasError) {
-                    templateMetadataView.progressIndicatorStep = templateMetadata.progressIndicatorStep;
-                    _sendMessage('setStep',templateMetadata.progressIndicatorStep);
+            model.getBatchMetadata().onMetadataUpdated.subscribe(function() {
+                var batchMetadataView = component.get('v.batchMetadata');
+                var batchMetadata = model.getBatchMetadata();
+                var headerChanged = Boolean(batchMetadataView.pageHeader !== batchMetadata.pageHeader);
+
+                batchMetadataView.labels = batchMetadata.labels;
+                batchMetadataView.mode = batchMetadata.mode;
+                batchMetadataView.hasError = batchMetadata.hasError;
+                batchMetadataView.errorMessage = batchMetadata.errorMessage;
+                batchMetadataView.pageHeader = batchMetadata.pageHeader;
+                batchMetadataView.pendingSave = batchMetadata.pendingSave;
+
+                if (!batchMetadataView.hasError) {
+                    batchMetadataView.progressIndicatorStep = batchMetadata.progressIndicatorStep;
+                    _sendMessage('setStep',batchMetadata.progressIndicatorStep);
                 } else {
                     component.find('notifLib').showNotice({
                         'variant': 'error',
                         'header': $A.get('$Label.c.PageMessagesError'),
-                        'message': templateMetadataView.errorMessage,
+                        'message': batchMetadataView.errorMessage,
                         closeCallback: function() {
                             //callback action here
                         }
                     });
                 }
 
-                if (templateMetadataView.mode === 'view') {
+                if (batchMetadataView.mode === 'view') {
                     component.set('v.isReadOnly', true);
-                } else if (templateMetadataView.mode === 'create' || templateMetadataView.mode === 'edit') {
+                } else if (batchMetadataView.mode === 'create' || batchMetadataView.mode === 'edit') {
                     component.set('v.isReadOnly', false);
-                    if (templateMetadata.mode === 'edit') {
-                        templateMetadata.labels.batchTemplateHeader = $A.get('$Label.c.bgeBatchTemplateEdit')
-                    } else if (templateMetadata.mode === 'create') {
-                        templateMetadata.labels.batchTemplateHeader = $A.get('$Label.c.bgeBatchTemplateNew');
+                    if (batchMetadata.mode === 'edit') {
+                        batchMetadata.labels.batchTemplateHeader = $A.get('$Label.c.bgeBatchEdit')
```

Comment:
We can also remove bgeBatchEdit and bgeBatchNew as labels, once this is removed

Reply:
In addition, we're not using the following custom labels anywhere, if you want to get rid of them at the same time (sound right @bethbrains?)

bgeBatchErrorRequiredAmount
bgeBatchErrorDefaultValue
bgeFLSError

Reply:
I am still using bgeFLSError. The other two can go.